util/metrics/src/service_metrics.rs

```diff
@@ -10,7 +10,8 @@ metrics about a gRPC server.
 For each method, the counters that are captured are:
 - num_req: number of requests
 - num_error: number of errors (can be used to calculate error rate)
-- num_status_code: number of GRPC status code (for more fine grained error rates/diagnostics)
+- num_status_code: number of gRPC status codes (to establish statistics on
+gRPC status codes, similar to how HTTP 2XX/4XX/5XX codes are profiled)
```

Comment:
Nit: indent
```suggestion
  gRPC status codes, similar to how HTTP 2XX/4XX/5XX codes are profiled)
```

Reply:
I feel like indents in a list like that leaves odd whitespace